app/lib/actions/katello/host/attach_subscriptions.rb

```diff
@@ -27,7 +27,7 @@ def plan(host, pools_with_quantities_params)
         end
 
         def finalize
-          ::Katello::Pool.where(:id => input[:pool_ids]).each(&:import_data)
+          ::Katello::Pool.where(:cp_id => input[:pool_ids]).map { |p| p.import_data(false) }
```

Comment:
any reason to change this to map?  .each() is likely better for this intent, since you're not saving the return values to a variable